uperf_wrapper/uperf_wrapper.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+import os
+
+from uperf_wrapper.trigger_uperf import Trigger_uperf
+
+
+class uperf_wrapper():
+
+    def __init__(self, parser):
+        parser.add_argument(
+            '-w', '--workload', nargs=1,
+            help='Provide XML workload location')
+        parser.add_argument(
+            '-r', '--run', nargs=1,
+            help='Provide the iteration for the run')
+        parser.add_argument(
+            '--resourcetype', nargs=1,
+            help='Provide the resource type for this uperf run - pod/vm/baremetal')
+        self.args = parser.parse_args()
+
+        self.args.server = ""
+        self.args.uuid = ""
+        self.args.user = ""
+        self.args.clientips = ""
+        self.args.remoteip = ""
+        self.args.hostnetwork = ""
+        self.args.serviceip = ""
+        self.args.server_node = ""
+        self.args.client_node = ""
+        self.args.cluster_name = "mycluster"
+
+        if "clustername" in os.environ:
+            self.args.cluster_name = os.environ["clustername"]
+        if "serviceip" in os.environ:
+            self.args.serviceip = os.environ['serviceip']
+        if "es" in os.environ:
```

Comment:
This part of code will be handled by the main code in run_snafu so we can remove this

Reply:
We will need to update uuid from it.

Reply:
uuid ? from es vars? I'm not sure I follow